pkgs/by-name/en/energyplus/package.nix

```diff
@@ -0,0 +1,51 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  python3,
+}:
+stdenv.mkDerivation rec {
+  pname = "energyplus";
+  version = "24.1.0";
+
+  src = fetchFromGitHub {
+    owner = "NREL";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-g8lWijcZvNkFclSiStU+7HWfm+F8LobP2kIJNV6zczE=";
+  };
+
+  buildInputs = [
+    cmake
+    python3
```

Comment:
At least cmake should be in nativeBuildInputs.

Reply:
Thanks for the overall review! This makes total sense as well. I wasn't thinking about this